scripts/validate-artifacts.ps1

```diff
@@ -109,6 +106,13 @@ if ($pip) {
     throw "Unable to find Public IP address resouce. Please create a Public IP resouce (Basic SKU, dynamic IP allocation) and try again."
 }
 
+if (($pip.sku.name -eq "Basic" ) -and ($pip.properties.publicIPAllocationMethod -eq "Dynamic")) { 
+    Write-Output "`u{2705} Checked Public IP SKU and allocation method - OK"
+} else { 
+    Write-Output `u{1F914}
+    Write-Warning "Unable to verify Public IP SKU and allocation method. Please check if public IP using 'Basic' SKU and dynamic IP allocation method."
+}
```

Comment:
The check for Public IP SKU and allocation method should use `throw` instead of `Write-Warning` to ensure consistency with other error checks in the script.